### ISSUE TITLE
✅ 🐞 Bug Report: Cannot reset link colour of links inside site settings. The color picker took a couple of seconds to appear.

### ISSUE THREAD

**Issue by Henzxon** [elementor/elementor#23365]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

I accidentally clicked the color picker inside the site settings for link colour and I can no longer reset it to default (blank) the colour picker pop-up takes a long while to appear and changes are no longer being applied.

### Steps to reproduce

1. Go to site settings > Typography > Link > Colour
2. Set the color to any global color
3. Manually select a new color value.  The colour picker pop-up) appears after a 3–5 seconds after or even longer.
4. Colour cannot be reset or changed anymore to default or blank.

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>Click to reveal</summary>

````txt


					
					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: Source distribution v5.7.39-42
	PHP Version: 7.4.33
	PHP Memory Limit: 768M
	PHP Max Input Vars: 3000
	PHP Max Post Size: 256M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.3
	Site URL: https://shannonb80.sg-host.com
	Home URL: https://shannonb80.sg-host.com
	WP Multisite: No
	Max Upload Size: 256 MB
	Memory limit: 256M
	Max Memory limit: 768M
	Permalink Structure: /%category%/%postname%/
	Language: en-AU
	Timezone: 0
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor Child
	Version: 2.0.0
	Author: Elementor Team
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 2.8.1
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: en_AU
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/115.0.0.0 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 3.15.2
		Author: Elementor.com

	Elementor Pro
		Version: 3.15.1
		Author: Elementor.com


== Elements Usage ==
	
	archive : 1
		archive-posts : 1
		container : 2
		theme-archive-title : 1
	footer : 1
		container : 6
		heading : 4
		icon : 1
		icon-list : 5
		nav-menu : 2
		share-buttons : 1
		text-editor : 1
		theme-site-logo : 1
	header : 1
		button : 1
		container : 2
		nav-menu : 1
		theme-site-logo : 1
	page : 1
		button : 8
		call-to-action : 1
		container : 11
		divider : 1
		form : 1
		heading : 10
		icon-box : 1
		share-buttons : 1
		spacer : 1
		text-editor : 1
	widget : 0
		button : 2
	wp-page : 9
		button : 10
		container : 43
		divider : 3
		form : 4
		heading : 29
		icon-box : 3
		icon-list : 2
		image : 4
		image-box : 3
		nested-accordion : 1
		posts : 1
		star-rating : 3
		text-editor : 19
	single-post : 1
		author-box : 1
		container : 5
		heading : 1
		post-comments : 1
		post-info : 1
		post-navigation : 1
		posts : 1
		progress-tracker : 1
		share-buttons : 1
		table-of-contents : 2
		text-editor : 1
		theme-post-content : 1
		theme-post-excerpt : 1
		theme-post-featured-image : 1
		theme-post-title : 1
	single-page : 1
		container : 2
		heading : 1
		theme-post-content : 1
	search-results : 1
		archive-posts : 1
		container : 3
		heading : 1
		search-form : 1
	error-404 : 1
		button : 1
		container : 2
		heading : 2
		image : 1
		text-editor : 1


== Settings ==
	
	cpt_support: post, page, e-landing-page
	disable_color_schemes: yes
	disable_typography_schemes: yes
	allow_tracking: yes
	unfiltered_files_upload: 1
	load_fa4_shim: yes


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Optimised DOM Output: Active
	Improved Asset Loading: Active
	Improved CSS Loading: Active
	Inline Font Icons: Active
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active by default
	Grid Container: Inactive by default
	Default to New Theme Builder: Active
	Hello Theme Header & Footer: Active by default
	Editor Top Bar: Inactive by default
	Landing Pages: Active by default
	Nested Elements: Active
	Lazy Load Background Images: Inactive by default
	Global Style Guide: Inactive by default
	Page Transitions: Active by default
	Notes: Active by default
	Loop: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default
	Menu: Inactive by default
	Taxonomy Filter: Inactive by default


== Log ==
	
Log: showing 20 of 332023-08-01 09:02:31 [info] elementor-pro::elementor_pro_updater Started 
2023-08-01 09:02:31 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-08-01 09:02:32 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-08-01 09:02:32 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.14.1',
  'to' => '3.15.0',
)]
2023-08-01 09:02:32 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.14.1',
  'to' => '3.15.0',
)]
2023-08-02 21:03:14 [info] elementor::elementor_updater Started 
2023-08-02 21:03:14 [info] Elementor/Upgrades - _on_each_version Start  
2023-08-02 21:03:15 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.15.0',
  'to' => '3.15.1',
)]
2023-08-02 21:03:15 [info] Elementor/Upgrades - _on_each_version Finished 
2023-08-02 21:03:15 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.15.0',
  'to' => '3.15.1',
)]
2023-08-09 10:01:27 [info] elementor::elementor_updater Started 
2023-08-09 10:01:27 [info] Elementor/Upgrades - _on_each_version Start  
2023-08-09 10:01:28 [info] Elementor/Upgrades - _on_each_version Finished 
2023-08-09 10:01:28 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.15.1',
  'to' => '3.15.2',
)]
2023-08-09 10:01:28 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.15.1',
  'to' => '3.15.2',
)]
2023-08-09 10:03:58 [info] elementor-pro::elementor_pro_updater Started 
2023-08-09 10:03:58 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-08-09 10:03:58 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-08-09 10:03:58 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.15.0',
  'to' => '3.15.1',
)]
2023-08-09 10:03:58 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.15.0',
  'to' => '3.15.1',
)]

JS: showing 6 of 6JS: 2023-07-24 11:43:32 [error X 1][https://shannonb80.sg-host.com/wp-admin/load-scripts.php?c=0&loadchunk_0=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=6.2.2:2:31823] Cannot read properties of undefined (reading 'value') 
JS: 2023-08-04 05:38:03 [error X 4][https://shannonb80.sg-host.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.15.1:3:603960] Cannot read properties of null (reading 'id') 
JS: 2023-08-04 05:57:16 [error X 3][https://shannonb80.sg-host.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.15.1:3:54034] Cannot read properties of undefined (reading 'activate') 
JS: 2023-08-04 06:26:49 [error X 10][https://shannonb80.sg-host.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.15.1:3:802938] Cannot read properties of undefined (reading 'localized') 
JS: 2023-08-08 14:56:04 [error X 60][https://shannonb80.sg-host.com/wp-content/plugins/elementor/assets/js/editor-modules.min.js?ver=3.15.1:3:7470] Cannot read properties of null (reading 'editor') 
JS: 2023-08-08 15:25:00 [error X 1][https://shannonb80.sg-host.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.15.1:3:318286] Cannot read properties of undefined (reading 'menu') 

PHP: showing 3 of 3PHP: 2023-08-04 05:23:47 [notice X 4][../wp-content/plugins/elementor/modules/history/revisions-manager.php::363] Trying to access array offset on value of type null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-08-05 02:09:14 [error X 1][../wp-content/plugins/elementor/core/wp-api.php::26] Uncaught Error: Call to undefined function Elementor\Core\get_plugins() in ../wp-content/plugins/elementor/core/wp-api.php:26
Stack trace:
#0 ../wp-content/plugins/elementor/modules/compatibility-tag/base-module.php(96): Elementor\Core\Wp_Api->get_plugins()
#1 ../wp-content/plugins/elementor/modules/compatibility-tag/base-module.php(58): Elementor\Modules\CompatibilityTag\Base_Module->get_plugins_with_header()
#2 ../wp-content/plugins/elementor/modules/compatibility-tag/module.php(57): Elementor\Modules\CompatibilityTag\Base_Module->get_plugins_to_check()
#3 ../wp-content/plugins/elementor/modules/compatibility-tag/base-module.php(148): Elementor\Modules\CompatibilityTag\Module->get_plugins_to_check()
#4 ../w [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-08-08 16:11:56 [warning X 6][../wp-content/plugins/elementor/app/modules/import-export/processes/export.php::286] Invalid argument supplied for foreach() [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==
	

			

````

</details>

**Comment by nicholaszein** [comment]
Hello @Henzxon! 

Thank you for submitting your issue! 🙏 

Please allow us some time to investigate it, and get back to you with our findings. 🧐 

Best regards

**Comment by VadymPogorelov** [comment]
This issue is similar to the one I described in issue [ColorPicker not working correctly in Page Settings](https://github.com/elementor/elementor/issues/22867)
That issue was closed, indicating that the problem was resolved in version 3.15.
I have Elementor 3.15.2 installed and the problem still remains

**Comment by Henzxon** [comment]
> This issue is similar to the one I described in issue [ColorPicker not working correctly in Page Settings](https://github.com/elementor/elementor/issues/22867) That issue was closed, indicating that the problem was resolved in version 3.15. I have Elementor 3.15.2 installed and the problem still remains

Yeah, I'm running the latest version and the issue still persist.

**Comment by Henzxon** [comment]
> This issue is similar to the one I described in issue [ColorPicker not working correctly in Page Settings](https://github.com/elementor/elementor/issues/22867) That issue was closed, indicating that the problem was resolved in version 3.15. I have Elementor 3.15.2 installed and the problem still remains

I found a workaround.

If you want to reset the to default settings, simply create a new global dummy font or color style, assign it to that specific settings and save changes. Then delete it afterwards. It will reset to default blank once deleted.

> **Just don't use any existing style to avoid data loss.**

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Core v3.15.3! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
